ssl/ssl_lib.c

```diff
@@ -4455,3 +4460,23 @@ int ssl_log_master_secret(SSL *ssl,
                           master_len);
 }
 
+int ssl_log_secret(SSL *ssl,
+                   const char *label,
+                   const uint8_t *client_random,
+                   size_t client_random_len,
+                   const uint8_t *secret,
+                   size_t secret_len)
+{
+    if (client_random_len != 32) {
+        SSLerr(SSL_F_SSL_LOG_MASTER_SECRET, ERR_R_INTERNAL_ERROR);
+        return 0;
+    }
+
+    return nss_keylog_int(label,
+                          ssl,
+                          client_random,
+                          client_random_len,
+                          secret,
+                          secret_len);
+}
```

Comment:
I agree that it could be merged with the function above.  Have a TLSv1.3 section and a non-TLSv1.3 one.